README.md

```diff
@@ -17,6 +17,36 @@
 ### Presentation (Kod.io 2014)
 [![angular-translate Talk](presentation2.png)](https://www.youtube.com/watch?v=C7xqaExvaQ4)
 
+### Getting started
+
+###Installing via Bower
+You can install the angular-translate package very easily using Bower. After installing Bower on your machine, simply run:
+
+`
+$ bower install angular-translate
+`
+
+This will install a package in your configured components folder. You can watch the bower package repository here. As you can see, it's pretty much broken down to things that matter. The raw source. For development as well as production use.
+
+###Installing with Git
+Another way to get the source of angular-translate, is to clone the whole repository from GitHub.
+
+`
+$ git clone git://github.com/PascalPrecht/angular-translate.git
+`
+
+You now have a full clone of the repository including the history and everything else that ever happened during the development of angular-translate. Do with the code what ever the fuck you want.
```

Comment:
Hm. The last sentence isn't really serious.
And on the other side, the whole git clone won't work for end users as cloning the source repository won't help you in using a distribution and you need (as partly documented below) a compilation step giving you the dist - version.
So to me, this is more something for the Developers rather then for the end users.